src/lesson25/js/modules/validation.js

```diff
@@ -53,63 +43,68 @@ export function removeErrorMessages(inputArr) {
   });
 }
 
-export function checkForUnfilled(input) {
-  const stateOfItem = getStateOfItem(input);
+export function checkForUnfilled(stateOfItem) {
   if (stateOfItem.empty) return true;
 }
```

Comment:
IMO
is**とするとbooleanが返ることがわかりやすいと思います！
あと、この関数はなくてもいい気がしました。
参照しているプロパティ名をisEmptyとかにするといいと思います
```suggestion
export function isEmpty(stateOfItem) {
  return stateOfItem.empty
}
```

Reply:
Thank you sae-san:)
I changed the function's name.
f7f6c7830ab33729e00066af3399cacf40747752